models/sdn-fabric-0.1.x/Makefile

```diff
@@ -47,7 +47,7 @@ mod-update: # @HELP Download the dependencies to the vendor folder
 	go mod tidy
 	go mod vendor
 
-image: mod-update openapi gnmi-gen # @HELP Build the docker image (available parameters: DOCKER_REPOSITORY, VERSION)
+image: mod-update openapi # gnmi-gen # @HELP Build the docker image (available parameters: DOCKER_REPOSITORY, VERSION)
```

Comment:
removing temporarily because it creates function name with `-` - need to adjust template